tests/e2e/CheckboxFilter.spec.ts

```diff
@@ -0,0 +1,52 @@
+import defaultFilter from "./CheckBoxFilter/pages/default.page";
+
+describe("CheckBoxFilter", () => {
```

Comment:
Tests are written like sentences. And lowest level should start with a Capitall

Reply:
`Check box filter`

Reply:
Changed